### PR TITLE
restructure optimization packages

### PR DESCRIPTION
**Pull request by maasencioh**
BREAKING CHANGE

add also levenberg-marquardt method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,13 +73,14 @@ require(['path/to/ml/dist/ml.min'], function (ML) {
...
     * Kernels: [`ML.Math.Kernel`](https://github.com/mljs/kernel)
-    * Optimization: [`ML.Math.Optimization`](https://github.com/mljs/optimization)
-    * BellOptimizer: [`ML.Math.BellOptimizer`](https://github.com/mljs/optimize-lorentzian)
 * Stat: [`ML.Stat`](https://github.com/mljs/stat)
     * Array: `ML.Stat.array`
     * Matrix: `ML.Stat.matrix`
     * Principal component analysis (PCA): [`ML.Stat.PCA`](https://github.com/mljs/pca)
     * Performance: [`ML.Stat.Performance`](https://github.com/mljs/performance)
+* Optimization: `ML.Optimization`
+    * BellOptimizer: [`ML.Math.BellOptimizer`](https://github.com/mljs/optimize-lorentzian)
```
Comment (targos) [inline README.md]: Math -> Optimization

File: README.md [hunk 2]
```diff
@@ -73,13 +73,14 @@ require(['path/to/ml/dist/ml.min'], function (ML) {
...
+    * LevenbergMarquardt: [`ML.Math.LevenbergMarquardt`](https://github.com/mljs/levenberg-marquardt)
```
Comment (targos) [inline README.md]: Same

**Comment by stropitek** [comment]
I think you forgot to add curve-fitting to the documentation

**Comment by maasencioh** [comment]
@stropitek it was intentional, the idea is to migrate this implementation to mine, in order to make it more readable and at the same take keep its performance

**Pull request merged**